blockstore/src/main/java/brooklyn/location/blockstore/VolumeManagers.java

```diff
@@ -68,11 +70,11 @@ public static VolumeManager newVolumeManager(Location location) {
             return new GoogleComputeEngineVolumeManager();
         } else if (provider.startsWith("openstack-nova")) {
             return new OpenstackVolumeManager();
+        } else if (provider.equals("vcloud-director")) {
```

Comment:
You can move these to common constants for both `VolumeManagers` and  `ExtraHddBodyEffector`